### DISCUSSION TITLE
Where does Chakra UI has its styles?

### DISCUSSION THREAD

**Discussion by georgekrax** [chakra-ui/chakra-ui#4863] [Q&A] [answered]
I am trying to find the code files where Chakra UI has its styling files, but I cannot find them anywhere😆. Any idea @segunadebayo ?

**Comment by daniel-schulz-codes** [comment, marked as answer]
Hej @georgekrax thanks for your question. The styles are defined in https://github.com/chakra-ui/chakra-ui/tree/main/packages/theme/src and are used from the components with the `useStyles` hook - does this answer your question?

**Reply by georgekrax** [reply]
Yeah, but the `<FormLabel />` for example, has `display: block` as I see in developer tools, but this css property is not defined in its component style file. Any explanation?

**Reply by daniel-schulz-codes** [reply]
Sometimes styles are as well defined on the component itself - which then could be overwritten by the theme if necessary. In the case of `FormLabel` you can find the declaration of `display:block` here: https://github.com/chakra-ui/chakra-ui/blob/main/packages/form-control/src/form-label.tsx#L50:L54

**Reply by georgekrax** [reply]
Oh yeah, my bad for not being more careful. Thank you very much for your response. But do you have also any idea, why chakra places classnames on elements, such as `chakra-input`, which do not have any styles attached to them?

**Reply by daniel-schulz-codes** [reply]
its for example for your convenience if you want to have either a selector or something to add extra styles with for example an external stylesheet - even though we encourage you to use internal styling

**Reply by georgekrax** [reply]
Okay. Also do you know how I can leverage the `css()` function from emotion through Chakra UI?
